gns3/qt/__init__.py

```diff
@@ -33,7 +33,7 @@
 import logging
 log = logging.getLogger("qt/__init__.py")
 
-from PyQt5 import QtCore, QtGui, QtNetwork, QtWidgets, Qt
+from PyQt5 import QtCore, QtGui, QtNetwork, QtWidgets, QtWebSockets, Qt
```

Comment:
QtWebSocket is imported below and shouldn't be imported here since it is not installed by default.